tests/JUnitOutputTest.cpp

```diff
@@ -499,7 +520,7 @@ TEST(JUnitOutputTest, testFailureWithDifferentFileAndLine)
 {
     testCaseRunner->start()
             .withGroup("testGroupWithFailingTest")
-                .withTest("FailingTestName").thatFails("Test failed", "importantFile", 999)
+            .withTest("FailingTestName", "sourcefile.c", 123).thatFails("Test failed", "importantFile", 999)
```

Comment:
This change isn't too good. It is breaking the builder. The way you would probably want to get this is:

.withTest("FailingTestName").in("sourcefile.c").lineNumber(123)

That way you do not need to pollute all the tests with the "sourcefile.c" information, which is irrelevant for most of the tests.

Making builders work like that is fun :)